### ISSUE TITLE
Contributor Approvals Should Not Be Considered

### ISSUE THREAD

**Issue by 0x4007** [ubiquity-os-marketplace/daemon-merging#22]
This doesn't make any sense to consider outsider approvals to automatically merge into our code, let alone default to 2. This logic should be removed. 

In theory it could be possible to run an org with a single member using this system but seems very unusual. I would remove this logic related to considering contributor approvals.  

https://github.com/ubiquity-os-marketplace/daemon-merging/blob/b9e0cf38c4bc11749f172b6212a35f95d26ab7ad/src/types/plugin-inputs.ts#L25

**Comment by gentlementlegen** [comment]
@Keyrxng It seems that the reminders are going wild with this issue, any idea about what's going on?

**Comment by YanVictorSN** [comment]
> @YanVictorSN, this task has been idle for a while. Please provide an update.

Currenty working on the QA tests.

**Comment by 0x4007** [comment]
Disqualify needs to close the pull

**Comment by gentlementlegen** [comment]
This would be the responsibility of https://github.com/ubiquity-os-marketplace/command-start-stop

**Comment by 0x4007** [comment]
I think this should be able to close pulls too. I'm not sure why it must be mutually exclusive. Besides that isn't on a clock like this one.

**Comment by gentlementlegen** [comment]
Yes that plugin was made to react to un-assignments so having it in the merging plugin would be redundant. However this seems not to work so I'l check what's going on.

**Comment by ishowvel** [comment]
I am waiting on reviews on both of them with qa's and tests both passing, only some refactoring suggestions were left

**Comment by ishowvel** [comment]
@0x4007 can I be assigned for this?

**Comment by ishowvel** [comment]
I'll wait until they get merged

**Comment by gentlementlegen** [comment]
https://github.com/ubiquity-os-marketplace/daemon-merging/pull/27 should be merged first to avoid major conflicts, since it adds the SDK to this plugin.
